src/feat/resample.h

```diff
@@ -185,6 +185,10 @@ class LinearResample {
   /// Resample(x, y, true) for the last piece.  Call it unnecessarily between
   /// signals will not do any harm.
   void Reset();
+
+  //// Return the input and output sampling rates (for checks, for example)
+  inline int32 GetInputSamplingRate() {return samp_rate_in_;};
```

Comment:
can you add spaces in side the {}?  I think the final semicolon is not required.